test/integration/uninstall/test_uninstall.py

```diff
@@ -306,3 +306,44 @@ def test_uninstall_tarantool_no_switch(tt_cmd, tmp_path):
         os.path.join(testdata_path, "bin", "tarantool_2.10.4"),
         os.path.join(testdata_path, "inc", "include", "tarantool_2.10.4")
     )
+
+
+@pytest.mark.parametrize("installed_version, version_to_uninstall", [
+    ("v1.2.3", "1.2.3"),
+    ("1.2.3", "v1.2.3")
+])
+def test_uninstall_tt_missing_version_character(
+        installed_version,
+        version_to_uninstall,
+        tt_cmd,
+        tmp_path):
+    configPath = os.path.join(tmp_path, config_name)
+    # Create test config.
+    with open(configPath, 'w') as f:
+        f.write(f'env:\n bin_dir: {tmp_path}\n inc_dir:\n')
+
+    # Copying built executable to bin directory,
+    # renaming it to version and change symlink
+    # to emulate that we have installed it like
+    # tt install tt v1.2.3.
+    # We don't really install it because bug we test
+    # is fixed only for current patch. Due to
+    # binary file is changing while running this
+    # bug won't appear only for current and future versions.
+    shutil.copy(tt_cmd, os.path.join(tmp_path, f"tt_{installed_version}"))
```

Comment:
Add a test that if there are two versions are installed: v123 and 123
The passed version only will be removed.

Reply:
How could it be? I thought we can have only one tag version installed using `tt install tt <tag>`: with v tag or without but not both.